payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheetActivity.kt

```diff
@@ -289,6 +291,7 @@ internal class PaymentSheetActivity : BaseSheetActivity<PaymentSheetResult>() {
         }
 
         viewBinding.googlePayButton.setOnClickListener {
+            updateErrorMessage()
```

Comment:
I wonder if it would make sense to have the setting of the error message happen in the viewModel rather than in the view.   Then you would clear it on checkout, and when selection changed?

Reply:
The issue is that there's more than one error message, here and below the GPay button. The logic for showing them is correct, but the logic for clearing them is what caused MOBILESDK-299 because of the `checkoutIdentifier`. So it's better that the Activity/Fragment clears the error message as soon as an action is taken, otherwise we would need a more complex logic on the ViewModel